python/ray/tune/tests/execution/test_actor_caching.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+import sys
+from ray.tune.tests.execution.utils import create_execution_test_objects, TestingTrial
+
+
+def test_actor_cached(tmpdir):
```

Comment:
Todo: I'll add more tests here (can be done in a follow-up)

Reply:
(this is primarily an example on how we can write Ray-less unit tests in the future)